_data/mentors.yml

```
@@ -774,6 +799,7 @@
     - linkedin: https://www.linkedin.com/in/tatiana-merkulova-6458a650/
 
 - name: Anushka Yadav
+  disabled: false 
```

```suggestion
  disabled: true 
```

You can add disabled to all mentors with zero hours.